package.json

```diff
@@ -38,7 +39,8 @@
     "test:watch": "nwb test-react --server",
     "predeploy": "npm run clean && npm run build",
     "deploy": "npm run predeploy && gh-pages -d demo/dist",
-    "lint": "node_modules/.bin/eslint . --fix"
+    "lint": "node_modules/.bin/eslint . --fix",
+    "prepare": "npm run build"
```

Comment:
I'm not so sure if this script is required.
It runs `build` after each `npm i` too, and I don't actually think a use-case for this.

Reply:
`prepare` script is used to install package direct from git repository.
If you don’t want this script. I will remove it.

Reply:
I didn't know that. Of course, it can stay then :)